examples/psp22/lib.rs

```diff
@@ -57,12 +72,7 @@ pub mod my_psp22 {
 
         #[ink(message)]
         pub fn set_hated_account(&mut self, hated: AccountId) {
```

Comment:
hmmm maybe we can add setters as well? just a `set` attribute and getter macro will implement set as well to `set` annotated fields